### DISCUSSION TITLE
Why goxel2?

### DISCUSSION THREAD

**Discussion by mdtrooper** [frokk/goxel2#10] [General]
I found this fork and it is awesome (because the original project is stopped).

But, why goxel2? and which are the differences between goxel and goxel2?

**Comment by pegvin** [comment]
ah firstly sorry for the late reply, seems like i am not getting notifications for discussions & issues.

secondly, goxel2 started because i saw stuff that can be improved like high resource usage, and stuff like that, currently i'm still adapting to the code so fixing high resource usage will take a while, but many additions are being made, like using Lua to procedurally generate art and in future even extend goxel2 using Lua.

The main aim of goxel2 is to improve and improve it, and any suggestions are accepted and might be even worked on.
